seed/migrations/0170_column_derived_column.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 3.2.13 on 2022-05-11 18:29
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('seed', '0169_auto_20220616_1028'),
```

Comment:
Is there any work to do to make previous existing columns work with this new migration?

Reply:
Yes. We would need to run a one-time script to create a related `column` for each `derived_column`. I believe this could be done by adding some logic to the migration.

Reply:
The updated migration will generate a `derived_column = None` attribute for each existing column, and each derived column will generate a related column. 

Users will have to reselect their derived columns in their column list profiles